backend/src/db/migrations/20200125010142-refactor_all_images_to_separate_type.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+  Refactor all our image properties on posts and users to a dedicated type
+  "Image" which contains metadata and image file urls.
+`
+
+const printSummaries = summaries => {
+  console.log('=========================================')
+  summaries.forEach(stat => {
+    console.log(stat.query.text)
+    console.log(JSON.stringify(stat.counters, null, 2))
+  })
+  console.log('=========================================')
+}
+
+export async function up() {
+  const driver = getDriver()
+  const session = driver.session()
+  const writeTxResultPromise = session.writeTransaction(async txc => {
+    const runs = await Promise.all(
+      [
+        `
+      MATCH (post:Post)
+      WHERE post.image IS NOT NULL
+      CREATE (post)-[:HERO_IMAGE]->(image:Image)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:33:20Z" title="Thursday, February 20th 2020, 8:33:20 pm +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+  Refactor all our image properties on posts and users to a dedicated type
+  "Image" which contains metadata and image file urls.
+`
+
+const printSummaries = summaries => {
+  console.log('=========================================')
+  summaries.forEach(stat => {
+    console.log(stat.query.text)
+    console.log(JSON.stringify(stat.counters, null, 2))
+  })
+  console.log('=========================================')
+}
+
+export async function up() {
+  const driver = getDriver()
+  const session = driver.session()
+  const writeTxResultPromise = session.writeTransaction(async txc => {
+    const runs = await Promise.all(
+      [
+        `
+      MATCH (post:Post)
+      WHERE post.image IS NOT NULL
+      CREATE (post)-[:TEASER_IMAGE]->(image:Image)
```


```suggestion
      CREATE (post)-[:HERO_IMAGE]->(image:Image)
```
I think we have started to rename this, so let's keep it consistent?